utils/sb2-config-gcc-toolchain

```diff
@@ -226,8 +226,8 @@ if [ $? != 0 ]; then
 	exit 1
 fi
 
-SBOX_CROSS_GCC_DIR=$(dirname $(which $GCC))
-if [ $GCC -v > /dev/null 2>&1 != 0 ]; then
+SBOX_CROSS_GCC_DIR="$(readlink -f $(dirname $(which $GCC)))"
```

Comment:
Quoting it this way is without effect I am afraid. Should it really do something, it would have to be:

```
SBOX_CROSS_GCC_DIR=$(readlink -f "$(dirname "$(which "$GCC")")")
```
(see how I intentionally omitted the top level double quotes - those are useless here as the `$(...)` is already treated as a single word.)

But since there is no quoting used elsewhere, I would leave it completely without quotes.